requirements.txt

```diff
@@ -1,22 +1,22 @@
-django>=2.0,<2.1
-django-registration-redux>=2.2,<2.3
-djangorestframework>=3.7,<3.8
-django-oauth-toolkit>=1.0,<1.1
-django-cors-headers>=2.1,<2.2
-requests>=2.18,<2.19
-django-filter>=1.1,<1.2
+django>=2.1,<2.2
```

Comment:
Because of that security vulnerability we got an alert for, maybe pin the lower version to 2.1.2 https://nvd.nist.gov/vuln/detail/CVE-2018-16984

Reply:
django>=2.1,<2.2 doesn't mean 2.1.1 is OK, it means "give me the highest version of 2.1 that is less than 2.2" So this should never resolve to 2.1.1. I'll change it anyway.

Reply:
Sure. I was thinking might as well be explicit. And also, I'd assume it would prevent us from getting those same vulnerability emails.